README.md

```diff
@@ -35,12 +35,18 @@ pip install deblur
 Example usage
 =============
 
-The input to deblur workflow is a directory of fasta files (1 per sample) or a demultiplexed FASTA or FASTQ file. The output is a biom table with sequences as the OTU ids (final.biom in the output directory).
+The input to deblur workflow is a directory of fasta files (1 per sample) or a demultiplexed FASTA or FASTQ file. The output are three biom tables (in the output directory) with sequences as the OTU ids:
 
-The simple use case just specifies the input fasta file (or directory) and output directory name:
+- all.biom : contains all deblurred reads.
+
+- positive-match.biom : contains only deblurred reads matching the positive filtering database (default is only 16S reads)
```

Comment:
@antgonza @josenavas @mortonjt @tanaes, can you comment please on what would be intuitive as filenames?